_posts/2022-06-20-notes-from-community-call.md

```diff
@@ -0,0 +1,50 @@
+
+* [Jan Ainali](https://publiccode.net/who-we-are/team/jan-ainali.html)
+* [Eric Herman](https://publiccode.net/who-we-are/team/eric-herman.html)
+* [Elena Findley-de Regt](https://publiccode.net/who-we-are/team/elena-findley-de-regt.html)
+
+## Updates from the Foundation
+
+* There's a [new version of the Standard](https://standard.publiccode.net/CHANGELOG.html): 0.3.0!
+* Our General Assembly had to be postponed by 2 weeks. This means that we're missing some of the steer we hoped we'd have by this call.
+
+## Discussion summary
+
+Our last in-person event was in October 2019; we’re optimistic we’ll be able to host an in-person event again this autumn. In this call, we brainstormed what sort of event we should hold, and for whom.
+
+### Event topic
+
+We started by discussing potential goals for an event. It quickly became clear that the codebase stewards think there would be both interest and value in hosting an event focused on obstacles to collaboration within codebase communities.
+
+We think it would be especially interesting to explore what people within codebase communities find scary about expanding their current collaborations - either nationally or internationally. We could then invite other people with relevant experience to share or even teach how they resolved those concerns.
+
+In our discussion on this call, we initially thought the event should focus on codebases getting their first international collaborators, since we've heard from many codebase communities that this change feels significant.
+
+We then considered that it might also make sense to add a second track for codebases just beginning in their collaborations, since any international expansion will depend on infrastructure that codebases will have to put into place for their first collaborators. See the [product assets for early incubation](https://about.publiccode.net/activities/codebase-stewardship/product-assets-for-early-incubation.html) for more about this. We thought attendees with international experience or ambitions might prove inspirational to newer codebase communities.
+
+### Invitees and logistics
+
+We then had a discussion about which specific codebase communities to potentially invite, either because they had told us of fears relate to scaling, or because they had unique and interesting experiences due to how they've grown. We thought people in a product owner role would benefit the most from this event.
+
+We also reviewed what we had liked most about our 2019 open house:
+
+* it was an interactive exploration of public code and the Foundation for Public Code (not just presentations, like the [FOSDEM public code devroom](https://fosdem.org/2022/schedule/track/public_code/))
+* participants found it valuable to meet and work with each other, not just Foundation for Public Code staff
+* it was intimate: it had 2 tracks including workshops, meaning a total size of ≈ 20 people (10 per track, since workshops of more than 10 people become unwieldy)
```

Comment:
```suggestion
* it was intimate: it had 2 tracks including workshops, meaning a total size of ≈20 people (10 per track, since workshops of more than 10 people become unwieldy)
```

Reply:
To make it consistent with the ≈30 later.